nixos/modules/services/networking/hylafax/modem-default.nix

```
@@ -16,7 +16,7 @@
   SessionTracing = "0x78701";
   UUCPLockDir = "/var/lock";
 
-  SendPageCmd = "${pkgs.coreutils}/bin/false"; # prevent pager transmit
-  SendUUCPCmd = "${pkgs.coreutils}/bin/false"; # prevent UUCP transmit
+  SendPageCmd = pkgs.lib.meta.getExe' pkgs.coreutils "false"; # prevent pager transmit
+  SendUUCPCmd = pkgs.lib.meta.getExe' pkgs.coreutils "false"; # prevent UUCP transmit
```

```suggestion
  SendPageCmd = lib.getExe' pkgs.coreutils "false"; # prevent pager transmit
  SendUUCPCmd = lib.getExe' pkgs.coreutils "false"; # prevent UUCP transmit
```
We should use lib directly from inputs

---

Done.